src/main/jbake/content/community/index.adoc

```diff
@@ -21,14 +21,15 @@ For more information about available support for TomEE please have a look on lin
 
 === Contribute to This Website
 
-This website is a simple JBake project you can find at https://git-wip-us.apache.org/repos/asf/tomee-site-generator.git.
```

Comment:
Why the user needs to know what is behind the scenes regarding infrastructure if he will never do a PR to asf repository? Contributors will only work with github. That will make users confused in my opinion as they will have 2 different sources.